include/caf/events/sensor_event.h

```diff
@@ -89,6 +89,29 @@ struct sensor_event {
 	struct event_dyndata dyndata; /**< Sensor data. Provided as floating-point values. */
 };
 
+/** @brief Set sensor period event.
+ *
+ * The set sensor period event can be submited by user to change sensors sampling period.
+ *
+ * The description field is a pointer to a string that is used to identify the sensor by the
+ * application. The Common Application Framework does not impose any standard way of describing
+ * sensors. Format and content of the sensor description is defined by the application.
+ *
+ * @note The set sensor period event related to the given sensor must use the same description as
+ *       #sensor_event related to the sensor.
+ */
+struct set_sensor_period_event {
+	/**< Event header. */
```

Comment:
`/**` instead `/**<` the second one is placed after the object being documented and the first one is placed before such object.